pkg/retry/retry.go

```diff
@@ -96,6 +96,10 @@ func isRetryable(err error) bool {
 			}
 		}
 		return true
+	case net.Error:
+		if e.Timeout() {
+			return true
+		}
```

Comment:
Hum… `net.Error` implementations might also implement `unwrapper` (at least `crypto/tls.permanentError`, maybe others, and in that case we might still want to follow `.Unwrap()`.

Reply:
Just do a fallthrough?